src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -226,6 +226,10 @@ bool OutputStateMachineEngine::ActionEscDispatch(const VTID id)
         success = _dispatch->HorizontalTabSet();
         TermTelemetry::Instance().Log(TermTelemetry::Codes::HTS);
         break;
+    case EscActionCodes::DECID_IdentifyDevice:
+        success = _dispatch->DeviceAttributes();
+        TermTelemetry::Instance().Log(TermTelemetry::Codes::DA);

```

Comment:
Note that I've reused the `DA` telemetry code here, since this is essentially an alias for the _Device Attributes_ report, but I'd be happy to create a separate telemetry code for it if that's preferable.

Reply:
meh, it's not like that's used for anything anyways 😛